examples/hard_funcs.py

```diff
@@ -1,10 +1,11 @@
 """problematic functions"""
```

Comment:
Here I get the following. Not sure if it is expected

```python-traceback
(base) czirwc1macos2701:magicgui nsofroniew$ python examples/hard_funcs.py
An error was encountered in the event loop:
 'functools.partial' object has no attribute '__name__'
```

Reply:
will remove from examples for now